crates/test-programs/build.rs

```diff
@@ -164,8 +199,30 @@ mod wasi_tests {
     cfg_if::cfg_if! {
         if #[cfg(not(windows))] {
             /// Ignore tests that aren't supported yet.
-            fn ignore(_testsuite: &str, _name: &str) -> bool {
-                false
+            fn ignore(testsuite: &str, name: &str) -> bool {
+                if testsuite == "wasi-tests" {
+                    match name {
```

Comment:
Right now trying to use any of these features (polling, file times, symlinks, or renaming) will pretty quickly hit an `unimplemented!()`. I don't really like asking to merge that, but it seems less confusing-and-subtle bug prone than `EINVAL` or whichever error. Very open to thoughts here.